### ISSUE TITLE
Radio buttons are sometimes not displaying their selected value when opening the property modal of a block.

### ISSUE THREAD

**Issue by mxx-lxg** [luyadev/luya-module-admin#324]
### What steps will reproduce the problem?
Create a block with radio buttons. Set a value and save. Re-open the property modal.

### What is the expected result?
When re-opened the radio buttons should display the previously selected value.

### What do you get instead? (A Screenshot can help us a lot!)
When re-opening the modal the radio buttons will display no selected value. However they work correctly otherwise. The selected value is saved and can be accessed in the block's view.

Edit: Looks like one radio button in the form will have its correct selected value. It appears that it is always the button group thats furthest down in the form. So if i have 3 radio button groups and select a value in group 1 it will show up. If i then select a value in group 2, that value will show up upon re-opening the modal and the value of group 1 will not. If i repeat that with group 3 that will show up.

### LUYA Check ouput (run this script and post the result: [luyacheck.php](https://gist.github.com/nadar/022f890e064c068e75ce6178cd14a09e))

`1: [ini_get('short_open_tag')] '1'
2: [ini_get('error_reporting')] '22527'
3: [phpversion()] '7.1.28'
4: [php_ini_loaded_file()] '/Applications/XAMPP/xamppfiles/etc/php.ini'
5: [php_sapi_name()] 'cgi-fcgi'
6: [isset($_SERVER['SERVER_SOFTWARE']) ? $_SERVER['SERVER_SOFTWARE'] : unknown] 'PhpStorm 2018.3.6'`
Had to comment out `apache_get_modules()`, because it threw an undefined error.

### Additional infos

| Q                | A
| ---------------- | ---
| LUYA Version     |  1.0 (admin module 1.2)
| PHP Version      | 7.1
| Platform         | XAMPP
| Operating system | OSX 10.14

**Comment by nadar** [comment]
Could you please provide the block config

**Comment by mxx-lxg** [comment]
There you go
```
        return [
            'vars' => [
                 ['var' => 'isFlat', 'label' => 'Flatrate', 'type' => self::TYPE_RADIO, 'options' => BlockHelper::radioArrayOption([
                     2 => 'in alle deutschen Netze',
                     1 => 'ja',
                     0 => 'nein'
                 ])],
                 ['var' => 'hasHotspot', 'label' => 'Hotspot', 'type' => self::TYPE_RADIO, 'options' => BlockHelper::radioArrayOption([
                     1 => 'ja',
                     0 => 'nein'
                 ])],
                ['var' => 'topTarif', 'label' => 'Top Tarif', 'type' => self::TYPE_RADIO, 'options' => BlockHelper::radioArrayOption([
                    1 => 'ja',
                    0 => 'nein'
                ])],
                 ['var' => 'style', 'label' => 'Style', 'type' => self::TYPE_RADIO, 'options' => BlockHelper::radioArrayOption([
                     'blue' => 'blau',
                     'green' => 'Grün',
                     'orange' => 'Orange',
                     'violett' => 'Violett'
                 ])],
                 ['var' => 'couponIncluded', 'label' => 'Gutscheinaktion', 'type' => self::TYPE_RADIO, 'options' => BlockHelper::radioArrayOption([
                     1 => 'ja',
                     0 => 'nein'
                 ])],
            ],
        ];
```

**Comment by nadar** [comment]
@TheMaaarc We had that situation already somewhere if i remember correctly?

**Comment by hbugdoll** [comment]
Same as luyadev/luya-module-cms#308.
Now fixed with #662.
